person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initilize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    if @age >= 18 || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] **Optional:** 
- Great work using the `public` access modifier, however, it is conventional to write all public methods before using the `private` access modifier.
- Moreso, the `@age >= 18 || @parent_permission`  would return a boolean, the `if-else` block is redundant.